src/app/components/MediaLoader/Placeholder/index.tsx

```diff
@@ -36,11 +36,11 @@ const MediaPlayerPlaceholder = ({
   const { service } = useContext(ServiceContext);
 
   return (
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
```

Comment:
Assuming this solves some screen reader / a11y weirdness ?

Reply:
Yea just prevents those lint errors so that the new placeholder behaves the same as the old one.